### PR TITLE
Add `nflverse_releases()`

### PR DESCRIPTION
**Pull request by mrcaseb**
a function that lists all available nflverse releases

### CODE REVIEW

**Comment by tanho63** [comment]
namingwise I think I prefer nflverse_download + nflverse_releases over download_nflverse and download_nflverse_releases

**Comment by mrcaseb** [comment]
> namingwise I think I prefer nflverse_download + nflverse_releases over download_nflverse and download_nflverse_releases

Yes I agree. Do you want me to change names in this PR?

**Comment by tanho63** [comment]
Yeah I'd go ahead and rename them here. I wonder if this releases function should do more, e.g. pull the timestamp for each release and/or pull a count of files (perhaps for each format)? (the former involves querying each release's timestamp.txt and the latter would be using datatable to summarise the output of `pb_list`)

File: pkgdown/_pkgdown.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ reference:
   - "load_pfr_advstats"
   - "load_snap_counts"
   - "load_contracts"
-  - "download_nflverse"
+  - "nflverse_download"
```
Comment (tanho63) [inline pkgdown/_pkgdown.yml:27]: nflverse_releases should appear next to wherever nflverse_download ends up, so either both as misc or both in this top section?

Comment (mrcaseb) [reply pkgdown/_pkgdown.yml:27]: I prefer top section here and changed it

**Comment by mrcaseb** [comment]
Ok, I added code that loads release data information and summarises them a bit. Seems to work as intended but we need to fix a check NOTE for an undefined global variable that is introduced with dtplyr.

**Review by tanho63 — APPROVED** [review]
As discussed LGTM

**Comment by mrcaseb** [comment]
Yeah I like the simplified version more, thanks!

**Pull request merged**